cranelift/isle/isle/src/compile.rs

```diff
@@ -7,6 +7,12 @@ use crate::{ast, codegen, sema, trie};
 pub fn compile(defs: &ast::Defs, options: &codegen::CodegenOptions) -> Result<String> {
     let mut typeenv = sema::TypeEnv::from_ast(defs)?;
     let termenv = sema::TermEnv::from_ast(&mut typeenv, defs)?;
+
+    // As the overlap checker currently finds a lot of overlap errors in the lowerings, require it
+    // to be explicitly enabled while we work through them.
+    #[cfg(feature = "check-overlap")]
+    crate::overlap::check(&mut typeenv, &termenv)?;
```

Comment:
I've changed it so that the feature controls whether or not the overlaps are considered errors. It currently only prints the number of overlaps when the feature is disabled, would you prefer that it always prints the errors out even if they're not going to fail the build?

Reply:
Nah, printing just the overlap count is fine, I think.